prov/efa/src/rdm/rxr_ep.c

```diff
@@ -1403,6 +1407,7 @@ int rxr_ep_init(struct rxr_ep *ep)
 			RXR_PKT_FROM_READ_COPY_POOL,
 			rxr_env.readcopy_pool_size,
 			rxr_env.readcopy_pool_size, /* max count */
+			RXR_128_BYTES_ALIGNMENT, /* Use 128 bytes alignment so it can support LL128 protocol */
```

Comment:
I think it is better to just use the number 128, along with the comment. No need for the extra macro.

Reply:
Updated.

Reply:
I did not see change though.

Reply:
can u refresh